src/main/java/frc/robot/JoysticksBindings.java

```diff
@@ -13,11 +15,15 @@ public class JoysticksBindings {
 
     private static final SmartJoystick FOURTH_JOYSTICK = new SmartJoystick(Ports.JoystickDriverStationPorts.FOURTH);
 
+    private static final KeyboardController KEYBOARD_CONTROLLER = new KeyboardController();
```

Comment:
keyboard bindings should not be in joystick bindings. please create bindings package and put in it JoystickBindings and KeyboardBindings